extensions/amp-video/1.0/video-wrapper.type.js

```diff
@@ -0,0 +1,44 @@
+/**
+ * Copyright 2020 The AMP HTML Authors. All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS-IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+/** @externs */
+
+var PlayerDef;
```

Comment:
Using `function():PreactDef.Renderable|string`, but now I'm wondering:

1. Should we define the props we expect on `function()`?
2. Is this otherwise an external type? Seems like it's the same type as the first argument for `React.createElement`.

Reply:
Hmm. Ok. That one already exists: `PreactDef.FunctionalComponent|string`. TAL and let's see if that fits.